electrum/submarine_swaps.py

```diff
@@ -1466,6 +1518,21 @@ async def get_pairs(self):
             self.sm.update_pairs(pairs)
             self.server_relays = content['relays'].split(',')
 
+    async def rebroadcast_event(self, event: Event, server_relays: Sequence[str]):
+        """If the relays of the origin server are different from our relays we rebroadcast the
+        event to our relays so it gets spread more widely."""
```

Comment:
note: nothing prevent us from rebroadcasting an event to a relay that already sent us that event.